### DISCUSSION TITLE
iCloud Photo Library not finished indexing

### DISCUSSION THREAD

**Discussion by carrfarm** [icloud-photos-downloader/icloud_photos_downloader#476] [Q&A] [answered]
I am getting this error on a couple different accounts and after waiting several minutes.
"iCloud Photo Library not finished indexing.  Please try again in a few minutes"

Any ideas?

**Comment by menkej** [comment, marked as answer]
Try again tomorrow and it will work.

Von meinem iPhone gesendet

**Comment by carrfarm** [comment]
Thank you, it does work now. I haven't seen that in the past. Is it related to iCloud changes or some change in icloudpd?  Will it happen very often?

**Comment by menkej** [comment]
iCloud „feature“ that appears usually when you first use the interface this project is using. Haven’t experienced that again after seeing that for the first time.

Von meinem iPhone gesendet
